samples/bluetooth/peripheral_esp/src/main.c

```diff
@@ -47,6 +47,52 @@
 #define ESS_EQUAL_TO_REF_VALUE			0x08
 #define ESS_NOT_EQUAL_TO_REF_VALUE		0x09
 
+/* ESS Measurement Descriptor – Sampling Functions */
+#define ESS_UNSPECIFIED 			0x00
```

Comment:
Suggest to add a prefix, like `ESS_DESC_SAMPLING_*` for this, and `ESS_DESC_APP_*` for the application descriptors. 

Right now you have `ESS_UNSPECIFIED` twice due to the lacking prefix distinction.

Reply:
I added prefixes to all of them including the triggers so that they would all follow the same pattern.